### PR TITLE
[ng] router example: show active router link

### LINKED ISSUE #369: [ng] router example UI: show active router link
This is point (4) from https://github.com/dart-lang/site-webdev/pull/283#issuecomment-278210839:
> 4. There's no indication, from the buttons, of which screen you're looking at. The screenshot, on the other hand, implies that the current screen's button is highlighted.

Contributes to #361.

### PR DESCRIPTION
**Pull request by chalin**
Fixes #369

Also gets rid of `#docregion` markers in UI when running the router example locally (and directly from the site-webdev repo — i.e., point (1) of https://github.com/dart-lang/site-webdev/pull/283#issuecomment-278210839).

**Staged at:** https://angulardart-org-dev2.firebaseapp.com

### CODE REVIEW

File: src/angular/guide/router/index.jade [hunk 1]
```diff
@@ -155,16 +155,23 @@ code-example(language="html").
 
   Consider the following template:
 
-+makeExcerpt('app/app.component.1.ts', 'template')
++makeExcerpt('app/app.component.1.ts (template and styles)', 'template')
```
Comment (kwalrath) [inline src/angular/guide/router/index.jade]: I don't see any styles in the output

File: src/angular/guide/router/index.jade [hunk 2]
```diff
@@ -155,16 +155,23 @@ code-example(language="html").
...
-//- TODO: mention routerLink: class.router-link-active here or possibly later in the details section below.
-//- TODO: mention `ActivatedRoute` and how to access it?
+//- TODO: `RouterLinkActive` is forthcoming in the new router. Once it is available, we'll be 
+//- able to make use of the a.active style in the global styles file.
+ The **`RouterLinkActive`** directive on each anchor tag helps visually distinguish the anchor for the currently selected _active_ route.
+
+:marked
+  The `RouterLink` directive also helps visually distinguish the anchor for the currently selected _active_ route.
+  The router adds the `router-link-active` CSS class to the element when the associated router link becomes active. 
+  As illustrated above, you can define this style alongside the template in the `@Component` annotation
```
Comment (kwalrath) [inline src/angular/guide/router/index.jade]: I don't see this illustrated above

File: src/angular/guide/router/index.jade (hunk 1 above)
Comment (chalin) [inline src/angular/guide/router/index.jade]: Did you look at the staged page? https://angulardart-org-dev2.firebaseapp.com/angular/guide/router ... I see it there.

If you are generating this locally, then you need to rebuild the site, or at least recreate the code fragments.

File: src/angular/guide/router/index.jade (hunk 2 above)
Comment (chalin) [inline src/angular/guide/router/index.jade]: Like for the previous comment, I see this in the staged page.

**Pull request merged**